consensus/parlia/parlia.go

```diff
@@ -53,13 +56,15 @@ const (
 	extraSeal        = 65 // Fixed number of extra-data suffix bytes reserved for signer seal
 	nextForkHashSize = 4  // Fixed number of extra-data suffix bytes reserved for nextForkHash.
 
-	validatorBytesLength = common.AddressLength
-	wiggleTime           = uint64(1) // second, Random delay (per signer) to allow concurrent signers
-	initialBackOffTime   = uint64(1) // second
-	processBackOffTime   = uint64(1) // second
+	validatorBytesLength           = common.AddressLength
```

Comment:
`validatorBytesLength` ==> `validatorBytesLengthBeforeBoneh`

`validatorBytesLengthAfterBoneh` ==> `validatorBytesLength`.

This fits out code style.

Reply:
accept